cmds/run.js

```diff
@@ -28,7 +28,7 @@ exports.handler = (argv) => {
     tjmFunctions.alreadyRegisteredCheck(jobContents)
         .catch((err) => {
             if (err) {
-                reply.warning(`Job ${jobContents.tjm.job_id} is already registered with cluster ${argv.c}`);
+                reply.warning(err);
             }
             return tjmFunctions.teraslice.jobs.submit(jobContents);
```

Comment:
I should have been more specific.  The desired behavior is that if the job has not been run before then it would register the job on the cluster, add the tjm data to the file, and run the job.  This commit breaks that work flow and new/ unregistered jobs end with the error 'Cannot read property 'id' of undefined'.